### PR TITLE
0004166: Engine Sounds of other Helicopters and planes missing...

### LINKED ISSUE #4166: Camera shoots multiple times if ammo > 1.
### Describe the bug

So, i discovered while creating a script that camera in MTA shoots multiple times if you press the button. It doesn't matter how much FPS client have and seems like other resources doesn't affect it. Also it doesn't matter if client have toggled "saving photos in GTA SA folder" - it will save one file of the first frame. Seems like resolution or window mode doesn't matter, just like any other video options. 

I have tested that on my own server and also with default MTASA resources (at least the ones that goes with MTA Map Editor in Testing mode (F5 button)).

It works fine if client will have 1 ammo available in camera, the sound and effect works as intended. 

I've got the video to show that issue: [YT video here](https://youtu.be/-yTQLIcpp30)

### Steps to reproduce

1. Join any server or Map Editor.
2. Take Camera.
3. Shoot it once or hold the attack button.
4. Try this with 1 ammo as well to see if it's fine.

### Version

Multi Theft Auto v1.6-release-23147

### Additional context

_No response_

### Relevant log output

```shell

```

### Security Policy

- [x] I have read and understood the Security Policy and this issue is not security related.

### PR DESCRIPTION
**Pull request by lopezloo**
It loads acceleration sound bank slot when remote ped enters vehicle. Similar to way the GTA does it for player (at `CAEVehicleAudioEntity::JustGotInVehicleAsDriver` / `0x4F5700`). This is limited only for planes and helis as we need.

I updated related structs & addresses. Credits to DK22Pac/plugin-sdk:
* https://github.com/DK22Pac/plugin-sdk/blob/26afb9cb0eb1f09bbb9ff82f680c85800b0b9bb8/plugin_sa/game_sa/CAEVehicleAudioEntity.h
* http://gtaforums.com/topic/194199-documenting-gta-sa-memory-addresses/?p=1064394505

### CODE REVIEW

File: Client/sdk/game/CAEAudioHardware.h [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+*  PROJECT:		Multi Theft Auto v1.0
+*  LICENSE:		See LICENSE in the top level directory
+*  FILE:		sdk/game/CAEAudioHardware.h
+*  PURPOSE:		Game audio hardware interface
+*
+*  Multi Theft Auto is available from http://www.multitheftauto.com/
+*
+*****************************************************************************/
+
+#ifndef __CGAME_CAEAUDIOHARDWARE
```
Comment (qaisjp) [inline Client/sdk/game/CAEAudioHardware.h]: use `#pragma once`

File: Client/sdk/game/CAEVehicleAudioEntity.h [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+*  PROJECT:		Multi Theft Auto v1.0
+*  LICENSE:		See LICENSE in the top level directory
+*  FILE:		sdk/game/CAEVehicleAudioEntity.h
+*  PURPOSE:		Vehicle audio interface
+*
+*  Multi Theft Auto is available from http://www.multitheftauto.com/
+*
+*****************************************************************************/
+
+#ifndef __CGAME_CAEVEHICLEAUDIOENTITY
```
Comment (qaisjp) [inline Client/sdk/game/CAEVehicleAudioEntity.h]: Use `#pragma once`

File: Client/game_sa/CAEAudioHardwareSA.h [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+*  PROJECT:     Multi Theft Auto v1.0
+*  LICENSE:     See LICENSE in the top level directory
+*  FILE:        game_sa/CAEAudioHardwareSA.h
+*  PURPOSE:     Audio hardware header
+*
+*  Multi Theft Auto is available from http://www.multitheftauto.com/
+*
+*****************************************************************************/
+
+#ifndef __CAEAUDIOHARDWARESA
```
Comment (qaisjp) [inline Client/game_sa/CAEAudioHardwareSA.h]: Use `#pragma once`

**Review by jushar — APPROVED** [review]
Good job, looks good to me.

**Pull request merged**